osquery/remote/serializers/json.cpp

```diff
@@ -21,7 +21,8 @@ Status JSONSerializer::serialize(const pt::ptree& params,
   try {
     pt::write_json(output, params, false);
   } catch (const pt::json_parser::json_parser_error& e) {
-    return Status(1, std::string("JSON serialize error: ") + e.what());
+    LOG(WARNING) << "Error serializing JSON: " << e.what();
```

Comment:
I'm worried about these callsites where the JSON error isn't very helpful until you know what is attempting to serialize the JSON. Perhaps the log should be done one frame above?